### PR TITLE
Update script for backporting.

### PR DESCRIPTION
**Pull request by KochetovNicolai**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix `auto-label` for backporting script.

### CODE REVIEW

File: utils/github/parser.py [hunk 1]
```diff
@@ -37,8 +41,20 @@ def _parse(self, text):
...
+            )
+
+            if stripped in category_headers:
                 next_category = True
 
         if category in Description.MAP_CATEGORY_TO_LABEL:
             self.label_name = Description.MAP_CATEGORY_TO_LABEL[category]
+        else:
+            if not category:
+                print('Cannot find category in pr description')
```
Comment (abyss7) [inline utils/github/parser.py]: Maybe print PR number too?

File: utils/github/parser.py [hunk 2]
```diff
@@ -10,6 +10,10 @@ class Description:
         'Performance Improvement': 'pr-performance',
         # 'Backward Incompatible Change': doesn't match anything
         'Build/Testing/Packaging Improvement': 'pr-build',
+        'Non-significant (changelog entry is not needed)': 'pr-non-significant',
```
Comment (abyss7) [inline utils/github/parser.py]: Maybe reuse `pr-other` tag? It's shorter.

File: utils/github/__main__.py [hunk 3]
```diff
@@ -80,6 +80,8 @@
 
 members = set(github.get_members("ClickHouse", "ClickHouse"))
 def print_responsible(pull_request):
+    if "author" not in pull_request or pull_request["author"] is None:
```
Comment (abyss7) [inline utils/github/__main__.py]: In which cases this may happen?

File: utils/github/parser.py (hunk 1 above)
Comment (KochetovNicolai) [reply utils/github/parser.py]: It is printed in message before. like
`Trying to auto-label pull-request: 8882`

File: utils/github/parser.py (hunk 2 above)
Comment (KochetovNicolai) [reply utils/github/parser.py]: I think that tag `pr-non-significant` is less surprising than `pr-other` while converting from this description.

File: utils/github/__main__.py (hunk 3 above)
Comment (KochetovNicolai) [reply utils/github/__main__.py]: I had a case when there was an exception and `pull_request["author"]` was `None`
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/nik-kochetov/dev/ClickHouse/utils/github/__main__.py", line 117, in <module>
    print(f'{CROSS_MARK} {bad["number"]}: {bad["url"]} ({print_responsible(bad)})')
  File "/home/nik-kochetov/dev/ClickHouse/utils/github/__main__.py", line 83, in print_responsible
    if pull_request["author"]["login"] in members:
TypeError: 'NoneType' object is not subscriptable
```
But I have lost exact pr :(

**Pull request merged**